features/old/services/multiservice.feature

```diff
@@ -6,7 +6,6 @@ Feature: Multiservice feature
   Background:
     Given a provider "foo.3scale.localhost"
     And a default service of provider "foo.3scale.localhost" has name "Fancy API"
-    And a service "Second service" of provider "foo.3scale.localhost"
```

Comment:
Why moving this down?

Reply:
because this line is added specific for that test only and is no useful for other tests